apps/nestjs-api/src/reminder-emails/reminder-emails.controller.ts

```diff
@@ -0,0 +1,116 @@
+import { Controller, Get, UseGuards } from '@nestjs/common'
+import { UserType } from '@talent-connect/common-types'
+import { CronjobAuthGuard } from '../auth/cronjob-auth.guard'
+import { ReminderEmailsService } from './reminder-emails.service'
+
+@Controller('reminder-emails')
+@UseGuards(CronjobAuthGuard)
+export class ReminderEmailsController {
+  constructor(private readonly reminderEmailsService: ReminderEmailsService) {}
+
+  @Get('/s3cr3t-3ndp01nt-t0-tr1gg3r-r3m1nd3r5/mentors-complete-profile')
+  async sendMentorCompleteProfileReminders() {
+    const mentorsWithDraftingProfile =
+      await this.reminderEmailsService.getDraftingConProfiles({
+        userType: UserType.MENTOR,
+      })
```

Comment:
We want to send this reminder to mentor profiles that were created 14 days ago AND have Drafting profile status. Do we also filter out the drafting mentor profiles by age?